utils/wasm-builder/src/cargo_command.rs

```diff
@@ -45,6 +46,7 @@ impl CargoCommand {
             target_dir: "target".into(),
             features: vec![],
             toolchain: Toolchain::nightly(),
```

Comment:
we still compile nightly

Reply:
Well I can rename it stable but later wasm-builder just uses `Toolchain::try_from_rustup`